src/viewer.js

```diff
@@ -328,6 +329,21 @@ $.Viewer = function( options ) {
 
     THIS[ this.hash ].prevContainerSize = _getSafeElemSize( this.container );
 
+    this._onViewerResize = onViewerResize;
```

Comment:
This was a good solution for the demo, but for the real patch I think we should pull it out.